src/coreclr/src/jit/lclmorph.cpp

```diff
@@ -0,0 +1,1137 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "jitpch.h"
```

Comment:
Looks like we're inconsistent about adding
```
#ifdef _MSC_VER
#pragma hdrstop
#endif
```
after this include. Must not be necessary (any more?).

Reply:
I noticed the inconsistency but I don't see what use `hdrstop` has, considering that we have `jitpch.clrjit.cpp`.